docs/config-file-ort-yml.md

```diff
@@ -153,7 +153,9 @@ curations:
     concluded_license: "Apache-2.0"
  ```
 
-For findings in third-party dependencies package-configurations can be used to replace findings:
+For findings in third-party dependencies package configurations can be used to replace findings if
+`enableRepositoryPackageConfigurations` is enabled in the evaluator section of `ort.conf`, see:
```

Comment:
The 'in the evaluator section' is incorrect as `enableRepositoryPackageCuration` and `enableRepositoryPackageConfigurations` both are set in the root level of `ort.conf` looking at the submitted change to `reference.conf`

Reply:
I would rephrase the sentence to align it more with [config-file-package-configuration-yml.md](https://github.com/oss-review-toolkit/ort/blob/master/docs/config-file-package-configuration-yml.md) wording e.g. "To overwrite scanner findings to correct identified licenses in a dependency a package configuration can be used. Note that this feature requires `enableRepositoryPackageConfigurations` to be enabled in the `ort.conf` see [reference.conf](../model/src/main/resources/reference.conf).

Reply:
Done. Added an `and` after findings.